libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestRequest.m

```diff
@@ -225,8 +225,17 @@ - (void)prepareRequestForSend
         // It's POSTish.  The Network SDK handles content-based requests more or less automatically,
         // but if you want to post a JSON object or other data, you have to manage the contents yourself.
         if (self.action.parameters.parameterStyle != CSFParameterStyleMultipart) {
-            // Standard POST data.  We'll assume we can just send it as JSON.
-            NSData *bodyData = [SFJsonUtils JSONDataRepresentation:self.queryParams];
+            NSData *bodyData = nil;
+            //Check if the POST should be sent as an encoded string rather than JSON
+            NSString* formEncoded = (NSString*)[self.queryParams objectForKey: @"x-www-form-urlencoded" ];
```

Comment:
There are far too many specialized content types to one-off each one in this area of the code.  It's what `[SFRestRequest setCustomRequestBodyString:contentType:]` and its other input analogs were made for.  See my previous comments.